### PR TITLE
fix some build issues

### PR DESCRIPTION
**Pull request by snecklifter**
I have no idea if this project is dead but it seems some build
issues need fixing so this is my first pass at doing so.

### CODE REVIEW

**Comment by snecklifter** [comment]
An attempt to fix basic issues in #76

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
+services:
+  - xvfb
 language: java
 
 jdk:
-  - oraclejdk8
-  - oraclejdk9
-  - openjdk7
+  - oraclejdk11
   - openjdk11
```
Comment (davidmoten) [inline .travis.yml:9]: add openjdk8 as well please

**Review by davidmoten — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (snecklifter) [reply .travis.yml:9]: The reason I removed these is because older versions fail with: `Expected feature release number in range of 9 to 17, but got: 8`

**Review by davidmoten — APPROVED** [review]

**Comment by snecklifter** [comment]
@davidmoten do we care about this tiny decrease in coverage?

**Comment by sylvestre** [comment]
No :)

**Pull request merged**